omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -121,4 +121,21 @@ public void shouldGetAConceptAttributeTypeByUuid() throws Exception {
     	Assert.assertEquals(conceptAttributeType.getUuid(), PropertyUtils.getProperty(result, "uuid"));
     	Assert.assertEquals(conceptAttributeType.getName(), PropertyUtils.getProperty(result, "name"));
     }
+
+    @Test
+    public void shouldRetireAConceptAttributeType() throws Exception {
+        final String UUID = "9516cc50-6f9f-11e0-8414-001e378eb67e";
+        ConceptAttributeType conceptAttributeType = service.getConceptAttributeTypeByUuid(UUID);
+        Assert.assertNotNull(conceptAttributeType);
+        Assert.assertFalse(conceptAttributeType.isRetired());
```

Comment:
Thanks @Muta-Jonathan . This is great!. Have you tried looking into using getRetired() as opposed to using isRetired()?
The later method seems to be deprecated. Could this [document](https://docs.openmrs.org/doc/org/openmrs/BaseOpenmrsMetadata.html) be helpful?

Reply:
Thank you so much @Irenyak1 for the wonderful review. Please keep it up! 👍